apps/bond/components/BondingProducts/Bonding/BondingList/BondingList.jsx

```diff
@@ -86,11 +90,17 @@ const getColumns = (onClick, isActive, acc, depositoryAddress, hideEmptyProducts
       dataIndex: 'fullCurrentPriceLp',
       key: 'fullCurrentPriceLp',
       width: 140,
-      render: (x, details) => (
-        <a href={details.currentPriceLpLink} rel="noopener noreferrer" target="_blank">
-          {x}
-        </a>
-      ),
+      render: (x, details) => {
+        if (isCurrentPriceLpZero(x)) {
```

Comment:
if the price is `0` then it will not stop loading
would nullish check be better for this?